vardautomation/comp.py

```diff
@@ -206,7 +206,7 @@ def extract(self, writer: Writer = Writer.PYTHON, compression: int = -1, force_b
                     '-i', 'pipe:', *outputs
                 ]
 
-                VideoEncoder(BinaryPath.ffmpeg, settings, progress_update=_progress_update_func).run_enc(clip, None, y4m=False)
+                VideoEncoder(BinaryPath.ffmpeg, settings, progress_update=_progress_update_func).run_enc(clip, None, y4m=False, prefetch=self.config.prefetch)
```

Comment:
There is no `config` in this class as pylint warned it.

Reply:
Implementing the previous [discussion](https://github.com/Ichunjo/vardautomation/pull/2#discussion_r723133502)'s result would also fix this, by just doing
```suggestion
                VideoEncoder(BinaryPath.ffmpeg, settings, progress_update=_progress_update_func).run_enc(clip, None, y4m=False, prefetch=self.prefetch)
```

Reply:
There is no `prefetch` parameter either. Beside, why would you manually set the prefetch in a comparison? I hope you don't have plan to make comparisons of insane filtering lol